RankSys-diversity/src/main/java/es/uam/eps/ir/ranksys/diversity/intentaware/reranking/AlphaXQuAD.java

```diff
@@ -84,15 +86,16 @@ public UserAlphaXQuAD(Recommendation<U, I> recommendation, int maxLength) {
         protected double nov(Tuple2od<I> iv) {
             return uam.getItemIntents(iv.v1)
                     .mapToDouble(f -> {
-                        return uam.pf_u(f) * uam.pi_f(iv, f) * redundancy.getDouble(f);
-                    }).sum();
+                        return uam.pf_u(f) * iam.pi_f(iv, f) * redundancy.getDouble(f);
```

Comment:
That is indeed more intuitive - pf_u comes from UserAspectModel and pi_f from ItemAspectModel.